src/ray/raylet/node_manager.cc

```diff
@@ -750,6 +750,12 @@ void NodeManager::ScheduleTasks() {
       EnqueuePlaceableTask(t);
     }
   }
+
+  // All remaining placeable tasks should be registered with the task dependency
+  // manager.
```

Comment:
Can you add a note here saying that it's ok to call `TaskPending` multiple times on the same task?